### ISSUE TITLE
Hyperparameter insights 

### ISSUE THREAD

**Issue by mfogelson** [ashawkey/stable-dreamfusion#59]
Hello, 

I have ran a couple of tests using this package and got the following results:

**Prompt:** "A DLSR of a cute dog" 
**Training Iters:** 50,000
![IMG_0207](https://user-images.githubusercontent.com/23368739/198284549-f4a34123-fa8b-4787-b8dc-7f4829b409d8.png) ![image](https://user-images.githubusercontent.com/23368739/198284689-f37f3a40-5040-4afd-a42c-d897ea97a36a.png)

**Prompt:** "A frog wearing a top hat" 
**Training Iters:**  100,000
![IMG_5911](https://user-images.githubusercontent.com/23368739/198285039-0d647778-da11-4ee7-901e-e179d2f40a93.png)
![image](https://user-images.githubusercontent.com/23368739/198285128-ad116b5c-6356-42c2-b837-d53d940aac15.png)

While it looks like the model is starting to produce useful outputs, I wonder if there are some key hyperparameters that can greatly increase the performance of the model (outside of just training iters):
1) Learning rate: I use the default 1e-3, but I notice that in the CLI output it is showing the lr = 1e-2
2) bg_radius: Does this have a large impact, what is the motivation for 1.4? 
3) jitter_pose: Does this only help the NERF or also the StableDiffusion
4) lambda_entropy: How does this impact the loss? Ive noticed the losses are very low while training ~10e-6

If people have thoughts / tests or model training checkpoints they have run please share!

**Comment by Tsingularity** [comment]
@mfogelson thanks for sharing the insights and results! they look really good.
I am just wondering that, based on your tuning experience, what do u think it's the optimal training iterations wrt training time and performance trade-off? does the model still improve after 50k iters?
Thanks!
